document/CoreFormat.mdk

```diff
@@ -1891,6 +1989,23 @@ a stable, opaque identifier for the rule.
 `"CA2101"`
 ~
 
+~ IsoNote
+Rule identifiers must be stable for two reasons:
+
+1. So specific checks can be disabled in build automation scripts, without the risks
```

Comment:
use 'referenced' instead of 'disabled' or say something like 'so specific checks can be referenced (e.g., to disable them)'